### ISSUE TITLE
No results for artists/tags which have + sign

### ISSUE THREAD

**Issue by bgilc** [Bionus/imgbrd-grabber#1391]
**Bug description**

Can't search artists/tags which have + sign. 

**Steps to reproduce**

1. Search for an artist who has + sign in name. Example: lin+ (https://danbooru.donmai.us/artists/11406)

**Expected behavior**

The plus sign is ignored. Grabber displays result for search of "lin" instead of "lin+".

**Context**

[main.log](https://github.com/Bionus/imgbrd-grabber/files/2299522/main.log)

**System information**

- OS: Windows 10 1703 Polish 
- Grabber version: 6.0.4

**Comment by Bionus** [comment]
Indeed, it would seem that Grabber doesn't properly URL-encode the tags in queries.
I just pushed a fix for this problem that should arrive in v6.0.5, thanks for the report! :+1:
